domains/buc-aa.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "AtlasL1",
+        "email": "loxingle1@gmail.com"
+    },
+    "record": {
+        "CNAME": "buc-aa.github.io"
```

Comment:
I cannot reach your website (404 page)

Reply:
What if the site is on https://buc-aa.github.io/.github?